crates/re_space_view_spatial/src/space_camera_3d.rs

```diff
@@ -56,18 +58,15 @@ impl SpaceCamera3D {
     pub fn project_onto_2d(&self, point_in_world: Vec3) -> Option<Vec3> {
         let pinhole = self.pinhole?;
         let point_in_cam = self.cam_from_world().transform_point3(point_in_world);
-        let point_in_image = pinhole.project(point_in_cam);
-        Some(point_in_image)
-    }
 
-    /// Unproject a 2D image coordinate as a ray in 3D space
-    pub fn unproject_as_ray(&self, pos2d: Vec2) -> Option<Ray3> {
-        let pinhole = self.pinhole?;
+        // View-coordinates are relevant here because without them we have no notion of what the image plane is.
+        // (it's not a given that e.g. XY is the camera image plane!)
```

Comment:
I don't think this is true. The projection model is defined to divide by Z and flattens the scene onto the XY plane. I am not aware of any conventions for a camera projection that uses any other plane.